web/src/components/Panels/ButtonPanel.styles.js

```diff
@@ -0,0 +1,8 @@
+export default () => ({
+  buttonPanel: {
+    padding: 20,
+    display: 'flex',
```

Comment:
Theres a mix of single quote and double quote strings in your PR. Hadnt noticed it before. the linter on the client forces single quotes on strings.